applications/chats/serializers.py

```diff
@@ -16,13 +16,24 @@ class Meta:
 class RoomSerializer(serializers.ModelSerializer):
     class Meta:
         model = Room
-        fields = ('id', 'created_at', 'topic', 'receiver', )
-        read_only_fields = ('id', 'created_at', 'topic', 'receiver', )
+        exclude = ('members', )
+        read_only_fields = ('id', 'created_at', 'topic', 'receiver', 'unread_message_count', )
 
     topic = serializers.ReadOnlyField(source='topic.name')
     receiver = serializers.SerializerMethodField()
+    unread_message_count = serializers.SerializerMethodField()
 
     def get_receiver(self, obj):
         current_user = self.context['request'].user
         receiver = obj.members.exclude(id=current_user.id).first()
         return UserSerializer(receiver).data
+
+    def get_unread_message_count(self, room):
+        current_user = self.context['request'].user
+        return (
+            room.message_set.filter(
+                is_read=False
+            ).exclude(
+                writer__id=current_user.id
+            ).count()
+        )
```

Comment:
I think the recommended by black would be:
```suggestion
        return (
            room.message_set.filter(is_read=False)
            .exclude(writer__id=current_user.id)
            .count()
        )
```